compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -374,12 +374,27 @@ setConvolutionVectorDistributionConfig(mlir::FunctionOpInterface entryPoint,
     intrinsics.emplace_back(mSize, nSize, kSize, aType, bType, cType);
   }
 
-  // Note that the following heuristic seeds are just placeholder values.
-  // We need to clean it up and make it adjusting to different targets.
-  // See https://github.com/openxla/iree/issues/16341 for details.
-  GPUMMAHeuristicSeeds seeds{/*bestSubgroupCountPerWorkgroup=*/4,
-                             /*bestMNTileCountPerSubgroup=*/8,
-                             /*bestKTileCountPerSubgroup=*/2};
+  GPUMMAHeuristicSeeds seeds;
```

Comment:
We need to make the change on the matmul configuration. This is inside the convolution one.